tests/test_query.py

```diff
@@ -32,31 +32,31 @@ def tearDown(self):
     def test_no_metadata_get_related_files(self):
         index = GDCIndexClient(uri=base_url)
 
-        results = index.get_related_files(uuids['small'])
+        results = index.get_related_files('small')
```

Comment:
Can you elaborate on this change? I think this method accepts a list of uuids.

Reply:
`uuids['small']` is a [dict](https://github.com/NCI-GDC/gdc-client/blob/a7f147c5ad4a60dc68784de7e3a7cc48f3f2bde6/tests/conftest.py#L46). I think in python2 this was fine (possibly dict types could be dictionary keys)?? But python3 doesn't allow this. See this [line](https://github.com/NCI-GDC/gdc-client/blob/a7f147c5ad4a60dc68784de7e3a7cc48f3f2bde6/gdc_client/query/index.py#L19) for what I mean... it expects a type string and then searches through a list of dictkeys

Reply:
Ah that makes sense. My mistake.